### PR TITLE
Milestone 3 : Add desktop version

### PR DESCRIPTION
**Pull request by jaflih**
Milestone 3: Add desktop version

- updated index.html
- updated portfolio.css
- added ou updated background images

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -206,43 +216,46 @@ <h4>Skills</h4>
...
-            <img class="contact_ind" src="./images/icons/home-indicator.png" alt="home-indicator">
-        </form>
+        <div class="contact_block_1">
+            <h2 class="common_title">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            <form class="contact_form" action="https://formspree.io/f/moqraknq" method="POST">
+                <input type="text"  name="name" class="contact_input" required placeholder="Full name" maxlength="30">
+                <input type="email" name="_replyto" class="contact_input" required placeholder="Email address" >
+                <textarea name="message" class="contact_textarea"  cols="30" rows="10" required  placeholder="Enter text here" maxlength="500"></textarea>
+                <input type="submit" class="contact_button button" value="Get It Touch">
+            </form>
```
Comment (MrBrN197) [inline index.html:226]: - [x] Your design is really good 👏👏. I noticed that your button is not aligned to the right. on the Desktop if you notice that is the case. Please make this change to keep in line with the Figma 👍.
## Figma Design
image

## Your Design 
image

- [ ] Notice also that the `textarea` on the Figma template has a green border. Please also make this change 🙏.

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
## Hi, @jihaneH 👋 

You've done a really nice job so far.
There are some issues that you still need to work on to go to the next project but you are almost there!
## To highlight 👏👏
- ✔ Your design is really great 👍. It matches the Figma really well.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (jaflih) [reply index.html:226]: Thank you for the feedback. I thought that it was green only when the mouse over it. I change it, and I change also the button place.

**Review by MrBrN197 — APPROVED** [review]
## Hi, @jihaneH  👏

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jaflih** [comment]
@MrBrN197
icegif-1433

**Pull request merged**